src/components/common/Tabs.js

```diff
@@ -17,19 +15,35 @@ export default function Tabs(props) {
   const onClickTabItem = tab => {
     setActiveTab(tab);
   };
+  /*
+   *  Comment[ here ]
```

Comment:
Comment here? What is the comment? Let's not commit it unless it's actually commented.

Reply:
Taking it out now,